bin/mailer_server.js

```diff
@@ -26,8 +26,6 @@ var mailerLog = require('../lib/senders/log')('mailer')
 var legacyMailerLog = require('../lib/senders/legacy_log')(mailerLog)
 var Mailer = require('../lib/senders/email')(legacyMailerLog)
 
-var dbConnect = require('../lib/senders/db_connect')()
```

Comment:
@philbooth do we need the mailer_server.js at all anymore? Who uses it? Need to remove it from here: https://github.com/mozilla/fxa-local-dev/blob/master/servers.json#L157
 
![image](https://user-images.githubusercontent.com/128755/35923518-78fdf528-0bee-11e8-89f6-f81649a7c584.png)

Reply:
> @philbooth do we need the mailer_server.js at all anymore? Who uses it?

Oh, maybe we don't, I didn't realise.

@jrgm / @jbuck, is there anything that uses `bin/mailer_server.js` in the auth server?

Reply:
Didn't want to make this change until we were sure about it, so I've opened #2289 as a follow-up to discuss in triage.